pkg/pluginmanager/manager.go

```diff
@@ -137,15 +137,15 @@ func discoverSpecificPluginGroups(pd []configtypes.PluginDiscovery, criteria *di
 }
 
 // DiscoverStandalonePlugins returns the available standalone plugins
-func DiscoverStandalonePlugins(criteria *discovery.PluginDiscoveryCriteria) ([]discovery.Discovered, error) {
+func DiscoverStandalonePlugins(criteria *discovery.PluginDiscoveryCriteria, options ...discovery.PluginDiscoveryOptions) ([]discovery.Discovered, error) {
 	discoveries, err := getPluginDiscoveries()
 	if err != nil {
 		return nil, err
 	} else if len(discoveries) == 0 {
 		return nil, errors.New(errorNoDiscoverySourcesFound)
 	}
 
-	plugins, err := discoverSpecificPlugins(discoveries, criteria)
+	plugins, err := discoverSpecificPlugins(discoveries, criteria, options...)
```

Comment:
There are two other places where `discoverSpecificPlugins()` is called, do we know that the options are only needed when calling `DiscoverStandalonePlugins()`?

Reply:
moved criteria into options